src/README.md

```diff
@@ -0,0 +1,66 @@
+# Introduction
```

Comment:
Should all this content be moved to the main `README.md` file so that we don't have duplicate readmes?

Reply:
I suppose that could work. Is there anything on the Readme that appears on GH that we don't want in the docs and vice versa?